programs/margin-orca/src/instructions/register_position_meta.rs

```diff
@@ -0,0 +1,110 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+//
+// Copyright (C) 2023 JET PROTOCOL HOLDINGS, LLC.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+use anchor_spl::token::{Mint, Token, TokenAccount};
+use jet_margin::{AdapterResult, PositionChange};
+
+use crate::*;
+
+#[derive(Accounts)]
+pub struct RegisterPositionMeta<'info> {
```

Comment:
```suggestion
pub struct RegisterMarginPosition<'info> {
```

I was initially confused by these four instruction names. I think it helps to specify which program's positions they impact.